group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -88,10 +93,12 @@ public class GroupMetadataManager {
     public static class Builder {
         private LogContext logContext = null;
         private SnapshotRegistry snapshotRegistry = null;
+        private Time time = null;
         private List<PartitionAssignor> assignors = null;
-        private TopicsImage topicsImage = null;
```

Comment:
Did we change this to MetadataImage because it was easier to pass in that class? Or do we need features in the metadata image that were not in the topic image?

Reply:
We actually need other features from the MetadataImage (e.g. metadata version).